### ISSUE TITLE
Not enough documentaion detail for child / parent remaining configuration

### ISSUE THREAD

**Issue by tomlut** [MindFreeze/ha-sankey-chart#48]
I've had to revert to the last version as I have no idea how this (3rd section) should now be configured. Not for the lack of trying. 
```
type: entities
title: Daily Energy Distribution
entities:
  - type: custom:hui-element
    card_type: custom:sankey-chart
    show_names: true
    height: 300
    round: 2
    unit_prefix: k
    wide: true
    sections:
      - entities:
          - entity_id: sensor.energy_from_grid_daily_peak
            name: Grid Peak Energy Used
            color: '#488fc2'
            children:
              - sensor.energy_imported_daily_total
          - entity_id: sensor.energy_from_grid_daily_offpeak
            name: Grid Offpeak Energy Used
            color: '#2f5f82'
            children:
              - sensor.energy_imported_daily_total
          - entity_id: sensor.energy_inverter_daily
            name: Solar Energy Produced
            color: '#ff9800'
            children:
              - sensor.self_solar_consumption_daily
              - sensor.energy_to_grid_daily
      - entities:
          - entity_id: sensor.energy_imported_daily_total
            name: Daily Imported Total
            children:
              - sensor.energy_consumed_daily_total
          - entity_id: sensor.self_solar_consumption_daily
            name: Self Consumed Solar
            color: '#ff9800'
            children:
              - sensor.energy_consumed_daily_total
          - entity_id: sensor.energy_to_grid_daily
            name: Solar Energy Exported
            color: '#8353d1'
      - entities:
          - entity_id: sensor.energy_consumed_daily_total
            name: Total Energy Used
            color: '#488FC2'
            remaining:
              name: Unmonitored
            children:
              - sensor.bar_fridge_daily_energy
              - sensor.cinema_av_daily_energy
              - sensor.comms_rack_daily_energy
              - sensor.dishwasher_daily_energy
              - sensor.electronics_bench_daily_energy
              - sensor.fridge_daily_energy
              - sensor.energy_downstairs_heat_pump_daily
              - sensor.energy_hot_water_daily_peak
              - sensor.energy_hot_water_daily_offpeak
              - sensor.led_table_daily_energy
              - sensor.lounge_av_daily_energy
              - sensor.lounge_dehumidifier_daily_energy
              - sensor.electric_blanket_daily_energy
              - sensor.mechanical_bench_daily_energy
              - sensor.pc_monitor_daily_energy
              - sensor.rack_fan_daily_energy
              - sensor.rumpus_dehumidifier_daily_energy
              - sensor.spb_electric_blanket_daily_energy
              - sensor.cinema_subwoofer_daily_energy
              - sensor.energy_upstairs_heat_pump_daily_peak
              - sensor.energy_upstairs_heat_pump_daily_offpeak
              - sensor.washing_machine_daily_energy
              - sensor.workshop_chargers_daily_energy
      - entities:
          - entity_id: sensor.bar_fridge_daily_energy
            name: Bar Fridge
            color: '#039BE5'
          - entity_id: sensor.cinema_av_daily_energy
            name: Cinema AV Gear
            color: '#039BE5'
          - entity_id: sensor.comms_rack_daily_energy
            name: Comms Rack
            color: '#039BE5'
          - entity_id: sensor.dishwasher_daily_energy
            name: Dishwasher
            color: '#039BE5'
          - entity_id: sensor.electronics_bench_daily_energy
            name: Electronics Bench
            color: '#039BE5'
          - entity_id: sensor.fridge_daily_energy
            name: Fridge
            color: '#039BE5'
          - entity_id: sensor.energy_downstairs_heat_pump_daily
            name: Downstairs Heat Pump
            color: '#039BE5'
          - entity_id: sensor.energy_hot_water_daily_peak
            name: Hot Water (Peak)
            color: '#039BE5'
          - entity_id: sensor.energy_hot_water_daily_offpeak
            name: Hot Water (Offpeak)
            color: '#039BE5'
          - entity_id: sensor.led_table_daily_energy
            name: LED Table
            color: '#039BE5'
          - entity_id: sensor.lounge_av_daily_energy
            name: Lounge AV Gear
            color: '#039BE5'
          - entity_id: sensor.lounge_dehumidifier_daily_energy
            name: Lounge Dehumidifier
            color: '#039BE5'
          - entity_id: sensor.electric_blanket_daily_energy
            name: Master Bedroom Elect Blanket
            color: '#039BE5'
          - entity_id: sensor.mechanical_bench_daily_energy
            name: Mechanical Bench
            color: '#039BE5'
          - entity_id: sensor.pc_monitor_daily_energy
            name: PC Monitor
            color: '#039BE5'
          - entity_id: sensor.rack_fan_daily_energy
            name: Rack Fan
            color: '#039BE5'
          - entity_id: sensor.rumpus_dehumidifier_daily_energy
            name: Rumpus Dehumidifier
            color: '#039BE5'
          - entity_id: sensor.spb_electric_blanket_daily_energy
            name: Spare Bedroom Elec Blanket
            color: '#039BE5'
          - entity_id: sensor.cinema_subwoofer_daily_energy
            name: Subwoofer
            color: '#039BE5'
          - entity_id: sensor.energy_upstairs_heat_pump_daily_peak
            name: Upstairs Heat Pump (Peak)
            color: '#039BE5'
          - entity_id: sensor.energy_upstairs_heat_pump_daily_offpeak
            name: Upstairs Heat Pump (Offpeak)
            color: '#039BE5'
          - entity_id: sensor.washing_machine_daily_energy
            name: Washing Machine
            color: '#039BE5'
          - entity_id: sensor.workshop_chargers_daily_energy
            name: Workshop Chargers
            color: '#039BE5'
    card_mod:
      class: inline-card-no-border
card_mod:
  class: top-level-card
```

**Comment by MindFreeze** [comment]
I just updated the 'Daily energy use' example in the README. Take a look

**Comment by tomlut** [comment]
Thank you, that does make it clearer. It was possibly one of the options I tried previously but it does not seem to be working:

![Untitled](https://user-images.githubusercontent.com/10679300/198209398-6ec7a23e-e07a-46c8-8489-031dd6fa0163.png)

```
type: entities
title: Power Distribution
entities:
  - type: custom:hui-element
    card_type: custom:sankey-chart
    show_names: true
    height: 300
    round: 2
    unit_prefix: k
    wide: true
    sections:
      - entities:
          - entity_id: sensor.grid_power
            name: Solar Power
            color: '#e0b400'
            children:
              - sensor.solar_power_consumed
              - sensor.metering_power_supplied
      - entities:
          - entity_id: sensor.solar_power_consumed
            name: Solar Power Consumption
            color: '#e0b400'
            children:
              - sensor.total_power_consumed
          - entity_id: sensor.metering_power_supplied
            name: Solar Power to Grid
            color: '#8353d1'
          - entity_id: sensor.metering_power_absorbed
            name: Grid Power Consumption
            children:
              - sensor.total_power_consumed
            color: '#e45e65'
      - entities:
          - entity_id: sensor.total_power_consumed
            name: Total Power Consumption
            children:
              - sensor.known_loads
              - unknown_loads
            color: '#ff8000'
      - entities:
          - entity_id: sensor.known_loads
            name: Monitored Loads
            children:
              - sensor.bar_fridge_power
              - sensor.cinema_av_power
              - sensor.cinema_subwoofer_power
              - sensor.comms_rack_power
              - sensor.dishwasher_power
              - sensor.downstairs_heat_pump_power
              - sensor.electric_blanket_power
              - sensor.electronics_bench_power
              - sensor.fridge_power
              - sensor.hot_water_power
              - sensor.led_table_power
              - sensor.lounge_av_power
              - sensor.lounge_dehumidifier_power
              - sensor.mechanical_bench_power
              - sensor.pc_monitor_power
              - sensor.pi3b_power
              - sensor.rack_fan_power
              - sensor.rumpus_dehumidifier_power
              - sensor.spb_electric_blanket_power
              - sensor.upstairs_heat_pump_power
              - sensor.washing_machine_power
              - sensor.workshop_chargers_power
            color: '#039BE5'
          - entity_id: unknown_loads
            type: remaining_parent_state
            name: Unknown
      - entities:
          - sensor.bar_fridge_power
          - sensor.cinema_av_power
          - sensor.cinema_subwoofer_power
          - sensor.comms_rack_power
          - sensor.dishwasher_power
          - sensor.downstairs_heat_pump_power
          - sensor.electric_blanket_power
          - sensor.electronics_bench_power
          - sensor.fridge_power
          - sensor.hot_water_power
          - sensor.led_table_power
          - sensor.lounge_av_power
          - sensor.lounge_dehumidifier_power
          - sensor.mechanical_bench_power
          - sensor.pc_monitor_power
          - sensor.pi3b_power
          - sensor.rack_fan_power
          - sensor.rumpus_dehumidifier_power
          - sensor.spb_electric_blanket_power
          - sensor.upstairs_heat_pump_power
          - sensor.washing_machine_power
          - sensor.workshop_chargers_power
    card_mod:
      class: inline-card-no-border
card_mod:
  class: top-level-card
```

**Comment by tomlut** [comment]
It seems to only work for the last section (different chart):

![Untitled2](https://user-images.githubusercontent.com/10679300/198214706-efe458f4-752e-4f60-b447-8b1242d6fa27.png)

```
type: entities
title: Daily Energy Distribution
entities:
  - type: custom:hui-element
    card_type: custom:sankey-chart
    show_names: true
    height: 300
    round: 2
    unit_prefix: k
    wide: true
    sections:
      - entities:
          - entity_id: sensor.energy_from_grid_daily_peak
            name: Grid Peak Energy Used
            color: '#488fc2'
            children:
              - sensor.energy_imported_daily_total
          - entity_id: sensor.energy_from_grid_daily_offpeak
            name: Grid Offpeak Energy Used
            color: '#2f5f82'
            children:
              - sensor.energy_imported_daily_total
          - entity_id: sensor.energy_inverter_daily
            name: Solar Energy Produced
            color: '#ff9800'
            children:
              - sensor.self_solar_consumption_daily
              - sensor.energy_to_grid_daily
      - entities:
          - entity_id: sensor.energy_imported_daily_total
            name: Daily Imported Total
            children:
              - sensor.energy_consumed_daily_total
          - entity_id: sensor.self_solar_consumption_daily
            name: Self Consumed Solar
            color: '#ff9800'
            children:
              - sensor.energy_consumed_daily_total
          - entity_id: sensor.energy_to_grid_daily
            name: Solar Energy Exported
            color: '#8353d1'
      - entities:
          - entity_id: sensor.energy_consumed_daily_total
            name: Total Energy Used
            color: '#488FC2'
            children:
              - sensor.bar_fridge_daily_energy
              - sensor.cinema_av_daily_energy
              - sensor.comms_rack_daily_energy
              - sensor.dishwasher_daily_energy
              - sensor.electronics_bench_daily_energy
              - sensor.fridge_daily_energy
              - sensor.energy_downstairs_heat_pump_daily
              - sensor.energy_hot_water_daily_peak
              - sensor.energy_hot_water_daily_offpeak
              - sensor.led_table_daily_energy
              - sensor.lounge_av_daily_energy
              - sensor.lounge_dehumidifier_daily_energy
              - sensor.electric_blanket_daily_energy
              - sensor.mechanical_bench_daily_energy
              - sensor.pc_monitor_daily_energy
              - sensor.rack_fan_daily_energy
              - sensor.rumpus_dehumidifier_daily_energy
              - sensor.spb_electric_blanket_daily_energy
              - sensor.cinema_subwoofer_daily_energy
              - sensor.energy_upstairs_heat_pump_daily_peak
              - sensor.energy_upstairs_heat_pump_daily_offpeak
              - sensor.washing_machine_daily_energy
              - sensor.workshop_chargers_daily_energy
              - unmonitored
      - entities:
          - entity_id: sensor.bar_fridge_daily_energy
            name: Bar Fridge
            color: '#039BE5'
          - entity_id: sensor.cinema_av_daily_energy
            name: Cinema AV Gear
            color: '#039BE5'
          - entity_id: sensor.comms_rack_daily_energy
            name: Comms Rack
            color: '#039BE5'
          - entity_id: sensor.dishwasher_daily_energy
            name: Dishwasher
            color: '#039BE5'
          - entity_id: sensor.electronics_bench_daily_energy
            name: Electronics Bench
            color: '#039BE5'
          - entity_id: sensor.fridge_daily_energy
            name: Fridge
            color: '#039BE5'
          - entity_id: sensor.energy_downstairs_heat_pump_daily
            name: Downstairs Heat Pump
            color: '#039BE5'
          - entity_id: sensor.energy_hot_water_daily_peak
            name: Hot Water (Peak)
            color: '#039BE5'
          - entity_id: sensor.energy_hot_water_daily_offpeak
            name: Hot Water (Offpeak)
            color: '#039BE5'
          - entity_id: sensor.led_table_daily_energy
            name: LED Table
            color: '#039BE5'
          - entity_id: sensor.lounge_av_daily_energy
            name: Lounge AV Gear
            color: '#039BE5'
          - entity_id: sensor.lounge_dehumidifier_daily_energy
            name: Lounge Dehumidifier
            color: '#039BE5'
          - entity_id: sensor.electric_blanket_daily_energy
            name: Master Bedroom Elect Blanket
            color: '#039BE5'
          - entity_id: sensor.mechanical_bench_daily_energy
            name: Mechanical Bench
            color: '#039BE5'
          - entity_id: sensor.pc_monitor_daily_energy
            name: PC Monitor
            color: '#039BE5'
          - entity_id: sensor.rack_fan_daily_energy
            name: Rack Fan
            color: '#039BE5'
          - entity_id: sensor.rumpus_dehumidifier_daily_energy
            name: Rumpus Dehumidifier
            color: '#039BE5'
          - entity_id: sensor.spb_electric_blanket_daily_energy
            name: Spare Bedroom Elec Blanket
            color: '#039BE5'
          - entity_id: sensor.cinema_subwoofer_daily_energy
            name: Subwoofer
            color: '#039BE5'
          - entity_id: sensor.energy_upstairs_heat_pump_daily_peak
            name: Upstairs Heat Pump (Peak)
            color: '#039BE5'
          - entity_id: sensor.energy_upstairs_heat_pump_daily_offpeak
            name: Upstairs Heat Pump (Offpeak)
            color: '#039BE5'
          - entity_id: sensor.washing_machine_daily_energy
            name: Washing Machine
            color: '#039BE5'
          - entity_id: sensor.workshop_chargers_daily_energy
            name: Workshop Chargers
            color: '#039BE5'
          - entity_id: unmonitored
            type: remaining_parent_state
            name: Unmonitored
    card_mod:
      class: inline-card-no-border
card_mod:
  class: top-level-card
```

**Comment by MindFreeze** [comment]
Looks like a strange bug. I'll try to reproduce it

**Comment by danieleh9** [comment]
As I have mentioned on HA forums, for me it works fine on Energy (kWh) but when using it for Power (kW) the remaining_parent_state value seems like its maybe miscalculated.

See last posts in HA forum thread:
[https://community.home-assistant.io/t/anyone-using-the-sankey-chart-card/423125/49](url)

**Comment by tomlut** [comment]
Yeah that fits with what I am seeing above.

**Comment by tomlut** [comment]
Fixed in 1.1.1, thanks!
